react/features/dynamic-branding/reducer.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+import { ReducerRegistry } from '../base/redux';
+
+import { SET_USER_CUSTOM_DATA } from './actionTypes';
+
+/**
+ * The name of the redux store/state property which is the root of the redux
+ * state of the feature {@code dynamic-branding}.
+ */
+const STORE_NAME = 'features/dynamic-branding';
+
+const DEFAULT_STATE = {
+    backgroundColor: '',
```

Comment:
Please document which keys we support in the config.js comment.

Reply:
I have documented the keys in `config.js`.